src/cbmc/CMakeLists.txt

```diff
@@ -57,3 +57,19 @@ if(NOT WIN32)
     PATTERN "c*"
     PATTERN "g*")
 endif()
+
+# bash completion
+if(NOT WIN32)
+  add_custom_command(OUTPUT "${CMAKE_SOURCE_DIR}/scripts/bash-autocomplete/cbmc.sh"
+    COMMAND "${CMAKE_SOURCE_DIR}/scripts/bash-autocomplete/extract_switches.sh"
+    DEPENDS $<TARGET_FILE:cbmc>
+  )
+  add_custom_target(cbmc.sh ALL
+    DEPENDS "${CMAKE_SOURCE_DIR}/scripts/bash-autocomplete/cbmc.sh"
+  )
+  install(
+    FILES ${CMAKE_SOURCE_DIR}/scripts/bash-autocomplete/cbmc.sh
+    DESTINATION etc/bash_completion.d
```

Comment:
This folder doesn't exist in osX, at least not by default. Is this the correct one for that platform?